SPECS/cloud-init/add-mariner-distro-support.patch

```diff
@@ -359,9 +359,33 @@ index 6951a0e3..411065f9 100644
     # Other config here will be given to the distro class and/or path classes
     paths:
        cloud_dir: /var/lib/cloud/
+diff -ruN a/systemd/cloud-init-local.service.tmpl b/systemd/cloud-init-local.service.tmpl
+--- a/systemd/cloud-init-local.service.tmpl     2022-05-18 08:23:24.000000000 -0700
++++ b/systemd/cloud-init-local.service.tmpl     2022-08-23 16:23:57.854218334 -0700
+@@ -1,9 +1,7 @@
+ ## template:jinja
+ [Unit]
+ Description=Initial cloud-init job (pre-networking)
+-{% if variant in ["ubuntu", "unknown", "debian", "rhel" ] %}
```

Comment:
Which variant are we? The "unknown" one? I think we might not even need any of these checks, since we're building only for Mariner anyway.

Reply:
Yes, we are not any one of these variants, and we tried to add "mariner" in the list but it is not recognized so that this if condition is always skipped. So we removed this variant check here.